aten/src/ATen/SparseCOOTensorImpl.cpp

```diff
@@ -1,15 +1,15 @@
 #include <ATen/ATen.h>
-#include <ATen/SparseTensorImpl.h>
+#include <ATen/SparseCOOTensorImpl.h>
```

Comment:
For your sanity and your reviewer's sanity, you're going to want to split SparseTensorImpl rename to SparseCOOTensorImpl to its own PR. Use https://github.com/ezyang/ghstack/ to do this. You'll have to open a new stack of PRs to do this.

Reply:
(It will help keep you sane, because it will be a lot easier to land the subset of this PR which is just SparseTensorImpl to SparseCOOTensorImpl, and then you won't have to worry on being the rebase merge-conflict wheel forever)

Reply:
OK works. I'll just focus on getting this PR merged and then tackle the naming issues separately.